src/SourceIndexServer/wwwroot/styles.css

```diff
@@ -1097,6 +1109,10 @@ ol a, ol a:link, ol a:hover, ol a:focus, ol a:active,
         color: #aaa;
     }
 
+    .imagePlusMinus, .resultGroupHeader::before, .rA::before {
+        filter: invert(1);
+    }

```

Comment:
https://caniuse.com/css-filters says this is go for all current browsers except IE and Opera Mini, which are the two I usually make a point of not caring about.